lib/install.ps1

```diff
@@ -558,10 +553,10 @@ function dl_urls($app, $version, $manifest, $bucket, $architecture, $dir, $use_c
             Write-Host $fname -f Cyan -NoNewline
             Write-Host " ... " -NoNewline
             $null = mkdir "$dir\_tmp"
-            & $extract_fn "$dir\$fname" "$dir\_tmp"
-            Remove-Item "$dir\$fname"
+            & $extract_fn "$dir\$fname" "$dir\_tmp" $true
             if ($extract_to) {
                 $null = mkdir "$dir\$extract_to" -force
+                warn "'`$extract_to' is deprecated. If you maintain this manifest, please refer to the manifest reference docs."
```

Comment:
I would rather go with:
```suggestion
                warn "'extract_to' property is deprecated. If you maintain this manifest, please refer to the manifest reference docs."
```
Since maintainers and users do not care about internal variables (No need to care about hem). They should be familiar with propoerties.